extensions/syntax-fm/src/search.tsx

```diff
@@ -0,0 +1,51 @@
+import { Action, ActionPanel, Icon, List } from "@raycast/api";
+import { launchAppleMusic } from "./api/apple-music";
+import ShowNotesDetail from "./components/show-notes-detail";
+import { useEpisodes } from "./hooks/use-episodes";
+
+export default function Command() {
+  const { isLoading, data } = useEpisodes();
+
+  return (
+    <List isLoading={isLoading}>
+      {data &&
+        data.map((item) => {
+          return (
+            <List.Item
+              key={item.hash}
+              title={`${item.number}: ${item.title}`}
+              accessories={[{ date: new Date(item.date) }]}
+              actions={
+                <ActionPanel>
+                  {item.href && <Action.OpenInBrowser url={`https://syntax.fm/${item.href}`} />}
+                  <Action
+                    icon={Icon.Play}
+                    title="Play in Apple Music"
+                    onAction={async () => {
+                      await launchAppleMusic(item.url, item.title);
+                    }}
+                  />
+                  <ActionPanel.Section>
+                    {item.youtube_url && (
+                      <Action.OpenInBrowser
+                        icon={Icon.Video}
+                        url={item.youtube_url}
+                        title="Watch on YouTube"
+                        shortcut={{ modifiers: ["cmd", "shift"], key: "v" }}
+                      />
+                    )}
+                    <Action.Push
+                      icon={Icon.Document}
+                      title="View Show Notes"
```

Comment:
View or Show? 🙂

Reply:
Thank you for the review @pernielsentikaer

That's a tough one; the notes are actually called "Show Notes" and that's what listeners refer to them as, which is why I went for this wording to "View the Show Notes".

<img width="441" alt="image" src="https://github.com/raycast/extensions/assets/7691252/cfd659e6-25e5-49e3-89a3-dd3f7e163566">

Reply:
That makes a lot more sense now 🙂